common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2521,6 +2522,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("180s")
 
+  val PARTITION_SORTER_WARMUP_ENABLED: ConfigEntry[Boolean] =
```

Comment:
I think we should make it clear that how PARTITION_SORTER_WARMUP_ENABLED and WORKER_PARTITION_SORTER_PER_PARTITION_RESERVED_MEMORY configurations related, then user can know what the impact is when disable this.